libs/facilities/frontend/view/src/lib/components/browse/browse.page.ts

```diff
@@ -1,13 +1,36 @@
 import { CommonModule } from '@angular/common';
-import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import {
+	ChangeDetectionStrategy,
+	Component,
+	Input,
+	OnInit,
+	ViewEncapsulation,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { facilities, Facility } from '../facility';
 
 @Component({
 	selector: 'lib-browse',
 	standalone: true,
-	imports: [CommonModule],
+	imports: [CommonModule, IxModule, RouterLink],
+
 	templateUrl: './browse.page.html',
 	styleUrl: './browse.page.scss',
 	encapsulation: ViewEncapsulation.None,
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class XdBrowsePage {}
+export class XdBrowsePage implements OnInit {
+	@Input() facilities: Facility[] = facilities;
```

Comment:
1. facitilies should be `signals()`
2. after Input line break
3. thats `protected readonly facilities` . Here is why https://stackoverflow.com/questions/4913025/reasons-to-use-private-instead-of-protected-for-fields-and-methods
4. This is `@Input({required: true})`

Reply:
I think we will change it to signals when we actually have the data from the backend. I added a line break. Right now it is actually not required because of the default values I set in ngOnInit. The input is only used for the issue Page right now.